docs/style-guide.md

```diff
@@ -0,0 +1,47 @@
+# Style Guide
+
+When contributing to the documentation, please ensure you follow [the AsciiDoc recommended practices].
+They are not currently rigidly enforced.
+However, this may change in the future.
+
+## Key Terms, Jargon, and Terminology
```

Comment:
What is the intention of having that in a style guide?

Reply:
It's a common attribute across a number of style guides. It shows how terms must be spelled, which avoids any ambiguity.